lua/wikis/commons/MvpTable.lua

```diff
@@ -60,22 +66,24 @@ function MvpTable.run(args)
 		return
 	end
 
-	local output = mw.html.create('table')
-		:addClass('wikitable prizepooltable collapsed')
-		:css('text-align', 'center')
-		:css('margin-top', args.margin .. 'px')
-		:attr('data-opentext', 'place ' .. (args.cutafter + 1) .. ' to ' .. #mvpList)
-		:attr('data-closetext', 'place ' .. (args.cutafter + 1) .. ' to ' .. #mvpList)
-		:attr('data-cutafter', args.cutafter + (String.isNotEmpty(args.title) and 1 or 0))
-		:attr('data-definedcutafter', '')
-		:node(MvpTable._mainHeader(args))
-		:node(MvpTable._subHeader(args))
-
-	for _, item in ipairs(mvpList) do
-		output:node(MvpTable._row(item, args))
-	end
-
-	return output
+	return HtmlWidgets.Table{
+		classes = {'wikitable', 'prizepooltable', 'collapsed'},
+		css = {
+			['text-align'] = 'center',
+			['margin-top'] = args.margin .. 'px'
+		},
+		attributes = {
+			['data-opentext'] = 'place ' .. (args.cutafter + 1) .. ' to ' .. #mvpList,
+			['data-closetext'] = 'place ' .. (args.cutafter + 1) .. ' to ' .. #mvpList,
+			['data-cutafter'] = args.cutafter + (String.isNotEmpty(args.title) and 1 or 0),
+			['data-definedcutafter'] = ''
+		},
+		children = WidgetUtil.collect(
+			MvpTable._mainHeader(args),
+			MvpTable._subHeader(args),
+			Array.map(mvpList, FnUtil.curry(MvpTable._row, args))
+		)
+	}
 end
 
 ---@class mvpTableParsedArgs
```

Comment:
should we move that up?
(we usually define anno classes below the imports but above the functions)